pipeline/pipeline_class.py

```diff
@@ -11,6 +11,8 @@
 
 import logging
 from pipeline_base import PipelineBase as Base
+import torch
```

Comment:
If we're importing torch here we should probably install it into the basic docker file